File: top-interview-questions-medium/Math/excel-sheet-column-number/171.py
class Solution:
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        arr = list(s)
        arr.reverse()
        res = 0
        for i, ele in enumerate(arr):
            res += 26 ** i * self.helper(arr[i])
        return res

    def helper(self, s):
        return ord(s) - 64

File: top-interview-questions-medium/Math/excel-sheet-column-number/171.js
/**
 * @param {string} s
 * @return {number}
 */
var titleToNumber = function(s) {
  const helper = s => s.charCodeAt() - 64
  const arr = s.split('').reverse()
  let res = 0
  for (let i = 0; i < arr.length; i++) {
    res += 26 ** i * helper(arr[i])
  }
  return res
}
